sunbird-platform-core/common-util/src/main/java/org/sunbird/common/responsecode/ResponseCode.java

```diff
@@ -796,6 +796,9 @@
       ResponseMessage.Key.ERROR_USER_MIGRATION_FAILED,
       ResponseMessage.Message.ERROR_USER_MIGRATION_FAILED),
   invalidIdentifier(ResponseMessage.Key.VALID_IDENTIFIER_ABSENSE,ResponseMessage.Message.IDENTIFIER_VALIDATION_FAILED),
+  mergeeIdRequired(ResponseMessage.Key.MERGEE_ID_MISSING, ResponseMessage.Message.MERGEE_ID_MISSING),
+  mergerIdRequired(ResponseMessage.Key.MERGER_ID_MISSING, ResponseMessage.Message.MERGER_ID_MISSING),
```

Comment:
use the existing code like mandatoryParamsMissing instead of defining new one.

Reply:
request contains only 2 inputs, so i think it is better to give exact name.